src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -88,6 +88,7 @@ object DatabaseConversions {
         blockAccountsToAccountRows.convert(from).map {
           _.into[Tables.AccountsHistoryRow]
             .withFieldConst(_.asof, Timestamp.from(from.timestamp.getOrElse(Instant.ofEpochMilli(0))))
+            .withFieldConst(_.isBaker, false)
```

Comment:
Is this actually needed? If the db-column defaults to false, I guess there's no need to explicitly set the value on the row scala type generated via slick

Reply:
you're, right good catch